.github/workflows/terraform-smoke.yml

```diff
@@ -0,0 +1,90 @@
+name: "Terraform Provider for Juju Smoke"
+on:
+  push:
+    branches: [2.9, 3.1, 3.2, 3.3, 3.4]
```

Comment:
Can we please change this to:
`[2.9, 3.*, 4.*, main]`

Reply:
I can change to `[2.9, 3.*]` this version of the workflow shouldn't run against 4.*, nor main at this time.

The terraform provider doesn't support them yet. We have to split from supporting 3.* and 2.9.last, to supporting 4.* and 3.last first.